app/uploaders/image_uploader.rb

```diff
@@ -0,0 +1,11 @@
+class ImageUploader < CarrierWave::Uploader::Base
+  if fog_credentials.present?
```

Comment:
We don't need to move that configuration because we have only one uploader, for now.